physics/drag_suite.F90

```diff
@@ -307,9 +307,10 @@ subroutine drag_suite_run(                                           &
    real(kind=kind_phys) ::  rcl, cdmb
    real(kind=kind_phys) ::  g_inv
 
-   real(kind=kind_phys), intent(out) ::                          &
+   real(kind=kind_phys), intent(inout) ::                        &
```

Comment:
Please check if this change is required, and if so, please check and change the corresponding intent information in `drag_suite.meta`, if necessary.

Reply:
Actually, in drag_suite.meta, the three variables dudt, dvdt and dtdt had always been designated "intent = inout".  This is the correct designation, and I was bringing the drag_suite.F90 intent information into agreement.